docker-compose.yml

```diff
@@ -90,6 +90,25 @@ services:
       - protectedplanet_bundler:/usr/local/bundle
     network_mode: host
 
+  api:
+    container_name: protectedplanet-api
+    build: ${API_ABSOLUTE_PATH}
+    command: /bin/bash -l -c "bundle exec rackup"
+    volumes:
+      - ${API_ABSOLUTE_PATH}:/ProtectedPlanetApi
+      - ${SSH_AUTH_SOCK}:${SSH_AUTH_SOCK}
+    network_mode: host
+    environment:
+      - SSH_AUTH_SOCK=${SSH_AUTH_SOCK}
+    env_file:
+      - ${API_ABSOLUTE_PATH}/.env
```

Comment:
what does this do?

Reply:
mapping the ssh agent that was passed in the environmental variable right before `docker compose up` - not needed for the api service

Reply:
both lines 99 and 102 were needed to make the ssh agent work inside the container